src/components/Pronouns.jsx

```diff
@@ -18,7 +18,7 @@ function getDiscordPronouns(userId, guildId) {
     })
   } else {
     const discordPronouns = getGuildMemberProfile(userId, guildId)?.pronouns || userProfile?.pronouns
-    return discordPronouns.replace(/[\s\u200b]+/g, ' ').trim()
+    return discordPronouns ? discordPronouns.replace(/[\s\u200b]+/g, ' ').trim() : discordPronouns // check for undefined
```

Comment:
Much easier way to fix this
```suggestion
    return discordPronouns?.replace(/[\s\u200b]+/g, ' ').trim()
```